### ISSUE TITLE
Defining rules for aggregation

### ISSUE THREAD

**Issue by gwpicard** [carbon-data-specification/Power-Systems-Data#19]
I'm not sure if this is necessary / desired at this stage, but I'm wondering if we should specify something around how to use the model to aggregate temporally / spatially.

For example, if we use the model to declare per-unit generation, should there be definitions of how to aggregate at the per-fuel level? Or per "zone" level?

Likewise for going from hourly --> yearly ... etc.

**Comment by m-mirz** [comment]
Yes, the aggregation method can make a big difference. I think it is important but probably not top priority for the first version?

**Comment by gwpicard** [comment]
Makes sense. I'll let you decide if you would want to keep this issue open in the meantime until we address this later

**Comment by gwpicard** [comment]
Re: conversation in #4, we consider something like this:

<img width="907" alt="Screenshot 2022-05-24 at 11 03 34" src="https://user-images.githubusercontent.com/7591224/169994083-0a0499c0-d6eb-4fff-8f97-555ab74e7329.png">
